README.md

```diff
@@ -36,6 +38,8 @@ Searched 14,621,463,323,674 nodes (86861 ms). TT: 1259040447 hits.
 Speed: 168.3B nps.
 ```
 
+In comparison, Stockfish 15 searches nodes at xyz million nps.
```

Comment:
We shouldn't compare with Stockfish. We shouldn't belittle any engine due to whatever reason. We should just promote ours as much as we can.